plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/block.tsx

```diff
@@ -106,18 +106,18 @@ const Block = ( {
 	return (
 		<div className={ clsx( 'wc-block-cart__submit', className ) }>
 			{ positionReferenceElement }
-			{ /* The non-sticky container must always be visible because it gives height to its parent, which is required to calculate when it becomes visible in the viewport. */ }
-			<div className="wc-block-cart__submit-container">
-				{ submitContainerContents }
-			</div>
-			{ /* If the positionReferenceElement is below the viewport, display the sticky container. */ }
-			{ positionRelativeToViewport === 'below' && (
+			{ /* Choose which container to display based on the position of the reference element */ }
+			{ positionRelativeToViewport === 'below' ? (
 				<div
 					className="wc-block-cart__submit-container wc-block-cart__submit-container--sticky"
 					style={ { backgroundColor } }
 				>
 					{ submitContainerContents }
 				</div>
+			) : (
```

Comment:
>Thanks for your suggestion, @tarunvijwani. As I've opted for a different approach, based on our discussion in p1718875553124459-slack-C8X6Q7XQU, I'd prefer to leave the logic as it is for now to prevent accidentally introducing a regression.

I still feel that using clsx we can make the code look simpler and easier to read. However, this is not a blocker, so I will approve the PR. 

```js
	// Determine if the container should be sticky based on the position relative to the viewport.
	const displayStickyContainer = positionRelativeToViewport === 'below';

	const submitContainerClass = clsx( 'wc-block-cart__submit-container', {
		'wc-block-cart__submit-container--sticky': displayStickyContainer,
	} );

	return (
		<div className={ clsx( 'wc-block-cart__submit', className ) }>
			{ positionReferenceElement }
			<div
				className={ submitContainerClass }
				style={ displayStickyContainer ? { backgroundColor }: {} }
			>
				{ submitContainerContents }
			</div>
		</div>
	);
	
```

Reply:
I've applied your suggestion to increase the readability, @tarunvijwani. Thanks for suggesting this. 🙌